NSS-Day-Cohort-42/rare-API-media-moguls
# Add tag

#### Changes Made
1. Added create method in post view file
1. within that method there is a loop to add posttags that were sent in array along with the post
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout addTag
```

3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> Try to create a post in postman
```
{
        "title": "Kitty Meow Town",
        "publication_date": "2020-11-20",
        "content": "These dogs don't even know where they are", 
        "category_id": 1,
        "image_url": "",
        "selected_tags": [
            {
            "id": 1,
            "label": "titansfan"
        },
        {
            "id": 2,
            "label": "ilovemydog"
        }
    ]
    }
```
>in postman run get all post tags to see if the post tag relationships were created for the tags that were in the selected tags array in the post object you created

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
